core/jobs.py

```diff
@@ -1133,24 +1133,21 @@ def get(cls, entity_id, strict=True):
             BaseRealtimeDatastoreClassForContinuousComputations, cls
         ).get(entity_id, strict=strict)
 
-    def put(self):
-        """Stores the current realtime layer entity into the database.
+    def _pre_put_hook(self):
+        """Operations to perform just before the model is `put` into storage.
 
         Raises:
             Exception. The current instance has an invalid realtime layer id.
-
-        Returns:
-            realtime_layer. The realtime layer entity.
         """
+        super(
+            BaseRealtimeDatastoreClassForContinuousComputations,
+            self)._pre_put_hook()
```

Comment:
This looks cleaner IMO, but up to you.
```suggestion
        super(
            BaseRealtimeDatastoreClassForContinuousComputations, self
        )._pre_put_hook()
```